src/util.js

```diff
@@ -202,6 +208,12 @@ module.exports = {
    * @return {String} String representation of the array
    */
   Uint8Array2str: function (bin) {
+
+    // Uncomment for debugging
+    if(!Uint8Array.prototype.isPrototypeOf(bin)) {
+      throw new Error('Uint8Array2str: Data must be in the form of a Uint8Array');
+    }
+
```

Comment:
@bartbutler did you want to keep these two validation checks in the code? It seems to me you commented them our during production, perhaps for performance?